recording_external_revenue/tests/test_raw_revenue_conversion.py

```diff
@@ -0,0 +1,17 @@
+# © 2020 - today Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import pytest
+from odoo.tests.common import SavepointCase
+from odoo.exceptions import ValidationError
+
+
+class TestMusicalArtworkDistribution(SavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.raw_revenue_1 = cls._make_raw_revenue()
+
+    def _make_raw_revenue(self, **kwargs):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/numigi/odoo-recording-addons/pullRequest?prid=5594863)